pkg/networkmanager/index.html

```diff
@@ -151,6 +161,33 @@ <h2 class="panel-title" translatable="yes">Unmanaged Interfaces</h2>
     </div>
   </div>
 
+  <script id="network-general-settings-template" type="x-template/mustache">
+    <form class="ct-form">
+      <label class="control-label" for="network-general-settings-name-input" translatable="yes">Name</label>
+      <input id="network-general-settings-name-input" class="form-control" type="text"
+             value="{{name}}"/>
+
+      <label class="checkbox-inline">
+        <input id="network-general-settings-autoconnect" type="checkbox" checked="false">
+        <span translatable="yes">Automatically connect to this network when it is available</span>
+      </label>
+
+      <div>
+        <span translatable="yes">Connection priority for auto connect</span>
+        <input id="network-autoconnect-priority-input" type="number" min="0"/>
```

Comment:
I have -999 as autoconnect-priority for many of my connections. I am not sure where that value comes from. In any case, negative numbers seem to be allowed just fine.